dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/TrackedEntityAttributeControllerTest.java

```diff
@@ -166,6 +167,102 @@ void getIndexableOnlyAttributes()
 
     }
 
+    @Test
+    void getIndexableOnlyAttributesWithDateAndTextValueType()
+    {
+
+        teaA.setValueType( ValueType.DATE );
+        manager.update( teaA );
+        teaB.setValueType( ValueType.TEXT );
+        manager.update( teaB );
+        JsonObject json = GET( "/trackedEntityAttributes?indexableOnly=true" ).content( HttpStatus.OK );
+        assertFalse( json.isEmpty() );
```

Comment:
Is `assertFalse( json.isEmpty() );` always done before assertAttributeList? If so please move it inside 😄